deploy/helm/sumologic/conf/opentelemetry-operator/instrumentation.cr.yaml

```diff
@@ -19,6 +19,10 @@ spec:
     - baggage
   resource:
     addK8sUIDAttributes: false
+  sampler:
+    # work around https://github.com/open-telemetry/opentelemetry-operator/issues/1886
+    # TODO: remove this after upgrading past 0.33.0
+    type: always_off
```

Comment:
This will disable generation of the spans on the instrumentation side. Please set it to `always_on` as default.